openfoodfoundation/openfoodnetwork
# Remove Deprecated delegate_belongs_to

#### What? Why?

`delegate_belongs_to` was [deprecated and removed from Spree Core ](https://github.com/spree/spree/pull/8275)in favor of native Rails `delegate`.

This will help make the code cleaner and easier to troubleshoot but also allows for easier detection of missing foreign keys. 

Came up in

- https://github.com/openfoodfoundation/openfoodnetwork/pull/11548

#### What should we test?

Test Gateways:
- Visit a store page
- Add items to cart
- Click on Cart
- Click Checkout
- Select Stripe or Paypal Express Payment Gateway
- Confirm Order

Test product import:
- New product
- New variant for existing product
- Update existing variant

#### Release notes



Changelog Category (reviewers may add a label for the release notes):




The title of the pull request will be included in the release notes.


#### Dependencies




#### Documentation updates
